canopen/node/remote.py

```diff
@@ -147,8 +147,16 @@ def __load_configuration_helper(self, index, subindex, name, value):
 
     def load_configuration(self):
         ''' Load the configuration of the node from the object dictionary.'''
+        # First apply PDO configuration from object dictionary
+        self.pdo.read(from_od=True)
+        self.pdo.save()
+
+        # Now apply all other records in object dictionary
         for obj in self.object_dictionary.values():
-            if isinstance(obj, ODRecord) or isinstance(obj, ODArray):
+            if 0x1400 <= obj.index < 0x1c00:
+                # Ignore PDO related objects
+                logger.debug(f"{obj.index:#06x}: {obj.name} already applied by pdo.save()")
```

Comment:
```suggestion
                logger.debug("0x%04X: %s already applied by pdo.save()", obj.index, obj.name)
```

See #443.  Do we really need this much debug logging though?

Reply:
Agree, it is something that would be logged every time so it may not add much value.

Reply:
Fair enough.  Removing.